src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -152,127 +226,126 @@ These variables can be set in your local environment shell before running and th
+The `test` command performs tests on your local instance of Che. For example, to check the ability to create a workspace, start the workspace by using a custom workspace runtime and then use it. For a list of all the tests available, use the `test` command.
 
-'''''
+*_upgrade_*
 
-_stop_
+The `upgrade` command manages the sequence of upgrading Che from one version to another. For a list of available versions that you can upgrade to, run the `che version` command.
 
-The default stop is a graceful stop where each workspace is stopped and confirmed shutdown before stopping system services. If workspaces are configured to snap on stop, then all snaps will be completed before system service shutdown begins. You can ignore workspace stop behavior and shut down only system services with –force flag.
+The Che upgrade is done by using a `eclipse/che:<version>` image that is newer than the version you currently have installed. For example, if you have 5.0.0-M2 installed and you want to upgrade to 5.0.0-M7:
 
-'''''
-
-_test_
-
-Performs some tests on your local instance of Che. It can for example check the ability to create a workspace, start the workspace by using a custom Workspace runtime and then use it. The list of all the tests available can be obtained by providing only `test` command.
-
-'''''
+. To get the new version of Che:
++
+----
+docker pull eclipse/che:5.0.0-M7
+----
++
+You now have two eclipse/che images (one for each version).
++
+. Use the new image to upgrade the old installation:
++
+----
+docker run <volume-mounts> eclipse/che:5.0.0-M7 upgrade
+----
 
-_upgrade_
+The upgrade command has numerous checks to prevent you from upgrading Che if the new image and the old version are not compatiable. For the upgrade procedure to proceed, the CLI image must be newer than the value of '/instance/che.ver'.
 
-Manages the sequence of upgrading Che from one version to another. Run `che version` to get a list of available versions that you can upgrade to.
+Following is a list of actions that the upgrade process performs in the background:
 
-Upgrading Che is done by using a `eclipse/che:<version>` that is newer than the version you currently have installed. For example, if you have 5.0.0-M2 installed and want to upgrade to 5.0.0-M7, then:
+. Performs a version compatibility check.
 
-----
-# Get the new version of Che
-docker pull eclipse/che:5.0.0-M7
+. Downloads new Docker images that are needed to run the new version of Che.
 
-# You now have two eclipse/che images (one for each version)
-# Perform an upgrade - use the new image to upgrade old installation
-docker run <volume-mounts> eclipse/che:5.0.0-M7 upgrade
-----
+. Stops Che if it is currently running triggering a maintenance window.
```

Comment:
"running triggering"? Which one?

Reply:
Changed to "Stops Che if it is currently running and triggers a maintenance window.".